www/webapp/src/views/SignUp.vue

```diff
@@ -155,43 +196,62 @@
       GenericCaptcha,
       ErrorAlert,
     },
-    data: () => ({
-      valid: false,
-      working: false,
-      errors: [],
-      LOCAL_PUBLIC_SUFFIXES: LOCAL_PUBLIC_SUFFIXES,
+    data: function () {
+      const self = this;
+      return {
+        valid: false,
+        working: false,
+        errors: [],
+        LOCAL_PUBLIC_SUFFIXES: LOCAL_PUBLIC_SUFFIXES,
 
-      mdiDns: mdiDns,
-      mdiEmail: mdiEmail,
+        mdiDns: mdiDns,
+        mdiEmail: mdiEmail,
 
-      /* email field */
-      email: '',
-      email_rules: [v => !!email_pattern.test(v || '') || 'We need an email address for account recovery and technical support.'],
-      email_errors: [],
+        /* email field */
+        email: '',
+        email_rules: [v => !!email_pattern.test(v || '') || 'We need an email address for account recovery and technical support.'],
+        email_errors: [],
 
-      /* captcha field */
-      captchaID: null,
-      captchaSolution: '',
+        /* captcha field */
+        captchaID: null,
+        captchaSolution: '',
 
-      /* outreach preference */
-      outreach_preference: true,
+        /* outreach preference */
+        outreach_preference: true,
 
-      /* terms field */
-      terms: false,
-      terms_rules: [v => !!v || 'You can only use our service if you agree with the terms'],
+        /* terms field */
+        terms: false,
+        terms_rules: [v => !!v || 'You can only use our service if you agree with the terms'],
 
-      /* domain field */
-      domain: '',
-      domainType: null,
-      domain_rules: [v => !!v && !!domain_pattern.test(v) || 'Domain names can only contain letters, numbers, dots (.), and dashes (-), and must end with a top-level domain.'],
-      dyn_domain_rules: [v => !!v && v.indexOf('.') < 0 && !!domain_pattern.test(v + '.' + LOCAL_PUBLIC_SUFFIXES[0]) || 'Your domain name can only contain letters, numbers, and dashes (-).'],
-      domain_errors: [],
-    }),
+        /* domain field */
+        domain: '',
+        domainType: null,
+        domain_rules: [v => !!v && !!domain_pattern.test(v) || 'Domain names can only contain letters, numbers, dots (.), and dashes (-), and must end with a top-level domain.'],
+        dyn_domain_rules: [v => !!v && v.indexOf('.') < 0 && !!domain_pattern.test(v + '.' + LOCAL_PUBLIC_SUFFIXES[0]) || 'Your domain name can only contain letters, numbers, and dashes (-).'],
+        domain_errors: [],
+
+        /* dynDNS limitations */
+        computedHeight: 0,
+        limitationsAccepted: false,
+        limitationsAccepted_rules: [v => (self.domainType !== 'dynDNS' || !!v)],
+      }
+    },
     async mounted() {
       if ('email' in this.$route.params && this.$route.params.email !== undefined) {
         this.email = this.$route.params.email;
       }
       await this.initialFocus();
+
+      const el = this.$refs['limitations'];
```

Comment:
This doesn't look right. Have you tried using `v-if` on this DOM element instead?

Reply:
I've tried using `v-if`, but it makes the interface jump. UX is much smoother if the box is displayed with an expand animation. Unfortunately, CSS can't animate the `height` property from `0` to `auto`; if only works if the target height absolute value is provided. The workaround used here is commonly referenced e.g. on Stack Overflow for this problem.

How do you suggest to proceed? (Get rid of the animation?)